src/Razor/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X/test/RazorPageDocumentClassifierPassTest.cs

```diff
@@ -51,10 +51,9 @@ public void RazorPageDocumentClassifierPass_LogsErrorIfDirectiveNotAtTopOfFile()
                 length: 5 + Environment.NewLine.Length);
 
             var expectedDiagnostic = RazorExtensionsDiagnosticFactory.CreatePageDirective_MustExistAtTheTopOfFile(sourceSpan);
-            var content = @"
-@somethingelse
-@page
-";
+            var content = Environment.NewLine +
```

Comment:
This is fine. The alternative is to pass `ignoreLineEndingDifferences: true` to `Assert.Equal` if you don't particularly care about the ending.

Reply:
> This is fine. The alternative is to pass `ignoreLineEndingDifferences: true` to `Assert.Equal` if you don't particularly care about the ending.

In this case it's not the `Equals` for strings which is failing, it's the diagnostics span locations that fail because of the differences in Newline length 😄